src/assets/js/alma-widgets-inject.js

```diff
@@ -20,12 +20,15 @@ jQuery( document ).ready(
 
 		function getAmountElement() {
 			var amountQuerySelector = settings.amountQuerySelector;
-
 			if ( amountQuerySelector ) {
-				return document.querySelector( amountQuerySelector );
+				var amountsQueryItem = document.querySelectorAll( amountQuerySelector );
+				if ( amountsQueryItem.length === 0 ) {
+					return null;
+				}
+				// Usefull for variable products with sales price.
+				return amountsQueryItem[amountsQueryItem.length - 1];
```

Comment:
so returning the latest "`bdi`" should works (without adding another technical field for `variable_product_price_on_sale_query_selector`) ... alright ?

This will works if we are in LR (Left to Right) writing env...

But, what's about RL (Right to Left) ... or if a custom integration displays `on_sale_amount` and `stroked_amount` differently in the DOM ?

Reply:
I forgot the "right to left" case, will do it !

Reply:
done.